content-assets/wcag-act-rules/earl/equal-access.json

```diff
@@ -5201,6 +5211,19 @@
       "@type": "TestSubject",
       "source": "https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/23a2a8/32bfac8a98cc212aa7bf9151bf40f665a7f51696.html",
       "assertions": [
+        {
+          "@type": "Assertion",
+          "test": {
+            "title": "aria_accessiblename_exists:fail_no_accessible_name_image",
+            "isPartOf": [
+              "WCAG2:name-role-value",
+              []
```

Comment:
@daniel-montalvo I understand from [EARL Reporting Format – Test Criterion](https://github.com/w3c/wcag-act-rules/blob/main/content/report/earl.md#test-criterion) that `isPartOf` contains "strings, listing the WCAG success criteria that fail when the rule or procedure fails".

What is the purpose of this empty array in this context? This pattern seems new and is used multiple times in this pull request.

Reply:
Hi @remibetin

Indeed, this PR introduces many more occurrences of this empty array, but these are not the only ones. For example, see [equal-access.json](https://github.com/w3c/wcag-act-rules/blob/main/content-assets/wcag-act-rules/earl/equal-access.json#l3829), which apparently was last edited 4 June 2024. Is there any impact in the UI that these new instances are creating and I may be missing?

Reply:
>  Is there any impact in the UI that these new instances are creating and I may be missing?

Not that I am aware of, but I do not have in-depth knowledge of this part of the website.
From quick investigation, I can't find any use of `isPartOf` property in ACT Rules pages.

@WilcoFiers Are we missing something?

Reply:
As agreed with Shawn, I have raised a separate issue https://github.com/w3c/wcag-act-rules/issues/315